### ISSUE TITLE
Missing endpoint: "Get a release by tag name"

### ISSUE THREAD

**Issue by Taytay** [peter-murach/github#237]
(This gem is fantastic, and has saved us a ton of time).

It appears that it's missing this endpoint:
```
Get a release by tag name

Get a release with the specified tag. Users must have push access to the repository to view draft releases.

GET /repos/:owner/:repo/releases/tags/:tag
```

**Comment by peter-murach** [comment]
Thank you for submitting this issue. On my part, I'm happy to merge PR, review and release new version asap. Do you have time to add this endpoint?

**Comment by peter-murach** [comment]
Included in the latest release `v0.13.0` and can be accessed as `github.repos.releases.tags.get 'owner', 'repo', 'tag'`
